src/components/modals/AddOrEditMovieForm/index.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import styles from '../../../css-modules/modal.module.css';
+import stylesHeader from '../../../css-modules/header.module.css';
+
+// Components
+import Input from './Input';
+
+interface InitialState {
```

Comment:
it's just the same type as `Movie` in `[AddOrEditModal.tsx](https://github.com/jhonfmg7/home-task-02/pull/3/files#diff-c97e9f2916d9231ecc96b081e08f7c41b650e554a5052cfdad3e87c2767c5330)`. I guess that you can create a `types` directory to store and reuse some comment types between components

Reply:
Thank you, I resolved this by creating a new types folder to add interfaces